easybuild/easyconfigs/v/vitessceR/vitessceR-20230110-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'Bundle'
+
+name = 'vitessceR'
+version = '20230110'
+versionsuffix = '-R-%(rver)s'
+local_commit = '8c67168'
```

Comment:
Maybe you can put a small comment regarding the versioning of this package, since you seem to have assigned a date as version yourself.

The package seems to contain a version in the `DESCRIPTION` file, but I'm not sure if that would be suitable? Maybe not (probably not, considering there are no properly tagged releases...), in which case I agree in assigning it a date as version and putting a local commit to uniquely define the version from our side - but it's still good to put some remark in the EasyConfig that _we_ did the versioning on our side.

Reply:
good point there, done